perl-cross assumes the sources use xconfig.h with -DUSE_CROSS_COMPILE
With perl 5.20.0, it's no longer true. Regardless of what mainline perl
uses, let's stick with the old xconfig.h way for now.

--- a/perl.h
+++ b/perl.h
@@ -43,7 +43,11 @@
  */
 /* #define PERL_RC_STACK */
 
-#include "config.h"
+#ifndef USE_CROSS_COMPILE
+#  include "config.h"
+#else
+#  include "xconfig.h"
+#endif
 
 /* This fakes up using Mingw for locale handling.  In order to not define WIN32
  * in this file (and hence throughout the code that isn't expecting it), this
--- a/perl_langinfo.h
+++ b/perl_langinfo.h
@@ -4,7 +4,11 @@
 #ifndef PERL_LANGINFO_H
 #define PERL_LANGINFO_H 1
 
+#ifdef USE_CROSS_COMPILE
+#include "xconfig.h"
+#else
 #include "config.h"
+#endif
 
 #if defined(I_LANGINFO)
 #   include <langinfo.h>
